### DISCUSSION TITLE
Family Sharing for purchases Version

### DISCUSSION THREAD

**Discussion by Hellmudth** [MobiusSync/MobiusSync#92] [General]
Hi,
I have purchased Möbius Sync for my privat account. Now, I liked to use Möbius Syng also on my company iPhone, so I add the comapny apple id as a family share. Unfortunately, the restore purchase did not work, so do I have to buy it again, for every device I have?

**Comment by Hellmudth** [comment]
As there is no answer, may I ask, if anybody is able to have purchased unlimited app on 2 devices without buy a licence for each device?

**Reply by SimonPickup** [reply]
Implementing family sharing is surprising complex from a development perspective and not something we plan on doing.

But the in-app purchase applies to all devices with your Apple ID. Purchase on one device, and use Restore Purchases on the other device(s).

**Reply by Hellmudth** [reply]
Thank you for clarifying this behavior. 
Unfortunately, the company iPhone run a different Apple ID, so I can not restore purchases for that device.

**Comment by argus-core** [comment]
I see that the Pro version supports family sharing but the in-app purchased version doesn't. Is there a way to switch my license to the Pro version to avail of family sharing?

**Reply by argus-core** [reply]
@sjpickup could you take a look at this?

**Reply by SimonPickup** [reply]
App Developers don't have the ability to do this directly. You have to contact Apple yourself: https://support.apple.com/en-gb/HT204084

**Comment by scrapix** [comment]
Ran into the same problem today.... somewhat disappointing not being able to use family sharing...

**Comment by kams19** [comment]
Could you make it clear on the in-app purchase option that family sharing is not available?
This is really disappointing!

**Comment by SimonPickup** [comment]
This is about to be released: See https://github.com/MobiusSync/MobiusSync/issues/208
